winrt/lib/text/CanvasFontFace.abi.idl

```diff
@@ -76,12 +76,24 @@ namespace Microsoft.Graphics.Canvas.Text
     [version(VERSION)]
     typedef struct CanvasGlyph
     {
-        int Index;
+        UINT16 Index;
```

Comment:
I thought it was a mistake because DirectWrite uses uint16 for indices.
But now that you say it, I see GetGlyphIndices returning an array of Int.
My bad, I will change it back.